README.md

```diff
@@ -140,6 +142,224 @@ Please also use a service like pastebin to provide logs and refrain from uploadi
 [Contributor Code of Conduct](Code_of_Conduct.md)
 By participating in this project you agree to abide by its terms.
 
-## Licence
 
+## License
 Licenced under The MIT License.
+
+
+## Releases
```

Comment:
do not insert changelog here

Reply:
The reason for me doing this is that for the acceptance process the changelog is not included, while the README is. That is why a lot of add-ons have moved the detailed Changelog to the README. Also because the addon.xml is limited to 1500 characters we leave a shorter Changelog in the addon.xml.

If you still prefer the Changelog to be a separate file, that's fine for me. I basically implemented what we did elsewhere.

Reply:
yes thanks don't want a github home page of infinite length because of the changelog which in the end few people read

Reply:
That's why it is at the end. But I'll move it out, no problem.

Reply:
Fixed.